OracleWebLogic/samples/12213-domain-home-in-image/Dockerfile

```diff
@@ -1,6 +1,6 @@
-#Copyright (c) 2014-2018 Oracle and/or its affiliates. All rights reserved.
+#Copyright (c) 2014-2020 Oracle and/or its affiliates. All rights reserved.
```

Comment:
Format is "2014, 2020," -- it's a comma + space between 2014 and 2020 and then a comma after.

Reply:
I see "2014-2020" in a few other files too.  Let me update those.

Reply:
I hate to be that guy, but there is no comma after and we no longer use "All rights reserved either.". The correct format now is `Copyright (c) 2014, 2020 Oracle and/or its affiliates.`

Reply:
@Djelibeybi , do you have the link to Oracle's official guideline on the copyright statement?

Reply:
Yes, but it's an internal link, so please email me directly and I'll send it to you.